test/Translate/Ops/struct.p4

```diff
@@ -0,0 +1,111 @@
+// RUN: p4mlir-translate --typeinference-only %s | FileCheck %s
+
+struct P {
+    bit<32> f1;
+    bit<32> f2;
+}
+
+struct T {
+    int<32> t1;
+    int<32> t2;
+}
+
+struct S {
+    T s1;
+    T s2;
+}
+
+struct Empty {};
+
+// CHECK: !Empty = !p4hir.struct<"Empty">
+// CHECK: !PortId_t = !p4hir.struct<"PortId_t", _v: !b9i>
```

Comment:
Maybe it's more readable to move `struct PortId_t { bit<9> _v; }` to before these checks?